test/index.test.js

```diff
@@ -16,7 +18,8 @@ t.test('handler', async t => {
 
     await handler(helper.generateEvent({ bucketRegion: 'us-east-2', bucket: 'cars', key: 'file1.car' }))
     t.equal(t.context.sns.publishes.length, 1)
-    t.ok(t.context.sns.publishes.some(p => p.Message.includes('IndexerCompleted')), 'handler published IndexerCompleted event to SNS')
+    const [publish] = t.context.sns.publishes
+    assertIsIndexerCompletedPublish(t, publish)
```

Comment:
don't really agree to move assertion far from the function, also because it's used only once

Reply:
This assertion also got much more complicated and those checks would have significantly increased the lines of code in this function, so it seemed nice to put it behind a layer of abstraction